examples/common/python/config/config.py

```diff
@@ -27,9 +27,14 @@
 import re
 import toml
 from string import Template
-from utility.file_utils import find_file_in_path
+from utility.file_utils import find_file_in_paths
 
-__all__ = [ "ConfigurationException", "parse_configuration_files", "parse_configuration_file" ]
+__all__ = [ "ConfigurationException", 
+    "parse_configuration_files", 
+    "parse_configuration_file", 
+    "read_config_from_toml" ]
+
+TCFHOME = os.environ.get("TCF_HOME", "../../")
```

Comment:
for TCFHOME, the default is set to  2 directories up (../../) i.e. examples/common folder. Doesn't it has to be 4 levels up (../../../../)?

Reply:
Good catch Pankaj. I moved the statement(along with a function)  from another module. Though I have removed the default option altogether now from the os.environ.get method. That argument needs to be checked throughout. Relative paths can be misleading depending on the current working directory.